### DISCUSSION TITLE
Http Pfad und json beim Shelly pro 1pm

### DISCUSSION THREAD

**Discussion by iukbox** [hoylabs/OpenDTU-OnBattery#272] [General]
Hat den jemand im Einsatz als Power Meter?
Http://IP-Adresse/Status wie beim 3em funktioniert nicht -> Not found

**Comment by iukbox** [comment]
Falls es jemand mal brauchen sollte.
Der obige Http Aufruf ist für Geräte der 1. Generation, für Geräte der 2. Generation lautet er:
http://IP_ADR_DES_SHELLY/rpc/Shelly.GetStatus
Liefert als Ergebnis:
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/26631519/6553fbaa-38ca-483b-aa73-eea797f42719)


Der JSON Pfad ist: switch:0/apower

**Reply by OlSteph** [reply]
Hinweis dazu: Wenn der Pro 1PM analog zu meinem Pro 3EM funktioniert dann geht das nur ohne eingerichtetes Passwort auf dem Shelly.
Das könnte dann hier weiter führen:
https://github.com/helgeerbe/OpenDTU-OnBattery/issues/251

**Comment by Manos1966** [comment]
@iukbox Vielen Dank, ich werde es mir notieren.

...wobei ich gehe davon aus, dass du nicht gewagt hast, die 1PM nicht mit dem Stromzaehler anzuschliessen... :-D

**Reply by iukbox** [reply]
War ne Hilfestellung für jemanden in Spanien auf einem Campingplatz. Er ist wohl Elektrofachkraft und die haben einen Zähler der auch Einspeisung als Verbrauch rechnet, aber der Betreiber will nicht wechseln. Wobei ich die Konstellation immer noch nicht ganz verstanden hab, was der 1PM dann messen soll.
Er kam nur mit der Software nicht klar.
